.github/workflows/ci.yaml

```
@@ -60,6 +60,9 @@ jobs:
         run: |
           ./ci/install_kaldi.sh
       - name: test integration
+        env:
+          TEST_QUANTIZATION: ${{ !contains(["1.0.1", "1.1.0", "1.2.0"], matrix.pytorch-version) }}
```

```suggestion
          TEST_QUANTIZATION: ${{ !contains(['1.0.1', '1.1.0', '1.2.0'], matrix.pytorch-version) }}
```
https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#literals